.github/workflows/cicd.yaml

```diff
@@ -191,61 +191,84 @@ jobs:
       - uses: actions/upload-artifact@v2
         if: failure()
         with:
-           name: trivy-reports
-           path: trivy-reports
+          name: trivy-reports
+          path: trivy-reports
 
   integration-test:
     runs-on: ubuntu-latest
     needs: [build]
     strategy:
       fail-fast: false
       matrix:
-        integration-test-script: [
-          'integration-test.sh',
-          'preconfig-integration-test.sh',
-          'cosign-integration-test.sh',
-          'deployment-integration-test.sh',
-          'load-test.sh'
-        ]
+        integration-test-arg:
+          [
+            "regular",
+            "cosign",
+            "namespace-val",
+            "deployment",
+            "pre-config",
+            "stress-test",
```

Comment:
We still need to fix that, but since the stress-test runs way longer than the rest, can we move it up in the matrix such that it at least gets executed first?